### PR TITLE
MetaRPC + Client/Examples refactoring

### PR DESCRIPTION
**Pull request by xlab**
## MetaRPC

* Added examples on MetaRPC (part of Exchange API).

**MetaRPC** is a subset of methods that allow getting the info about server connection, such as health, backend version, network latency. Also allows subscribing to a stream that will suggest another endpoint if the server gracefully shutdowns.

## Bonus
Refactored the client and examples according to PEP 8. Fix for loosy typings of amounts.

### CODE REVIEW

**Comment by xlab** [comment]
`injective-exchange` latest `dev` required, specifically this commit (which fixes the streaming example)
https://github.com/InjectiveLabs/injective-exchange/commit/c27b7336cc8a25d066cc21e79c7febbb0dc4f946

File: pyinjective/constant.py [hunk 1]
```diff
@@ -78,36 +79,36 @@ def __init__(
...
         )
 
     @classmethod
     def testnet(cls):
         return cls(
-            lcd_endpoint='testnet.lcd.injective.dev',
-            grpc_endpoint='testnet-sentry0.injective.network:9900',
-            grpc_exchange_endpoint='testnet-sentry0.injective.network:9910',
+            lcd_endpoint='https://testnet.lcd.injective.dev',
+            grpc_endpoint='localhost:9900',
```
Comment (achilleas-kal) [inline pyinjective/constant.py:94]: Nam and Nikola recently added another sentry for testnet so we can include it here as a backup

File: pyinjective/utils.py [hunk 2]
```diff
@@ -22,60 +22,70 @@
...
-    return str(int(be_amount))
+    return int(be_amount)
+
 
-def floor_to(value: float, target: float) -> str:
+def floor_to(value: float, target: float) -> int:
     value = Decimal(str(value))
     target = Decimal(str(target))
-    result = Decimal(int(floor(value / target)) * target)
+    result = int(int(floor(value / target)) * target)
```
Comment (achilleas-kal) [inline pyinjective/utils.py:73]: Minor error caused by the mass refactoring, should be

result = int(floor(value / target)) * target

**Review by achilleas-kal — COMMENTED** [review]
Two minor errors caused from the mass refactoring, everything else looks great :)

File: pyinjective/constant.py (hunk 1 above)
Comment (xlab) [reply pyinjective/constant.py]: Oops, that's a typo, I changed to localhost only to test this out.

**Pull request merged**